apollo-router/src/configuration/cors.rs

```diff
@@ -1,28 +1,141 @@
 //! Cross Origin Resource Sharing (CORS configuration)
+//!
+//! This module provides configuration structures for CORS (Cross-Origin Resource Sharing) settings.
+//!
+//! # Default Behavior
+//!
+//! When the `policies` field is omitted from the CORS config, the router uses a default policy:
+//! - **Origins:** `["https://studio.apollographql.com"]`
+//! - **Methods:** `["GET", "POST", "OPTIONS"]`
+//! - **Allow credentials:** `false`
+//! - **Allow any origin:** `false`
+//!
+//! # Policy Configuration
+//!
+//! When specifying individual policies within the `policies` array:
+//! - **Origins:** Defaults to an empty list (no origins allowed) unless explicitly set
+//! - **Match origins:** Defaults to an empty list (no regex matching) unless explicitly set
+//! - **Methods:** Has three possible states:
+//!   - `null` (not specified): Use the global default methods
+//!   - `[]` (empty array): No methods allowed for this policy
+//!   - `[values]` (with values): Use these specific methods
+//! - **Allow headers:** Defaults to an empty list (mirrors client headers) unless explicitly set
+//! - **Expose headers:** Defaults to an empty list unless explicitly set
```

Comment:
If a global value is set for either of these two, then they will default to that global value, right?

Reply:
[ce968cb](https://github.com/apollographql/router/pull/7853/commits/ce968cb524da0d7e059af7b23914747c3cd6074a)

Reply:
@bryncooke addressed this in https://github.com/apollographql/router/pull/7853/commits/ce968cb524da0d7e059af7b23914747c3cd6074a